nav2_util/src/lifecycle_utils.cpp

```diff
@@ -24,20 +24,46 @@ using std::string;
 namespace nav2_util
 {
 
-void bringupLifecycleNode(const std::string & node_name)
+#define RETRY(fn, retries) \
```

Comment:
There are other places in the code besides life-cycle that could benefit from this retry method.  For example for Auto Localization currently I'm using the BT retry node for this purpose but we might need a simple retry method for other tasks.

Reply:
I agree. I'll do that as a separate PR. It seems like a big enough change that it should have a focused review and not just tacked on as an addendum to this one.